Robot2019/src/main/java/frc/robot/OI.java

```diff
@@ -75,9 +75,9 @@
     hatchEjectBtn.whenPressed(new ToggleHatchEject(hp));
 
     cargoIntakeBtn = new JoystickButton(manipulator, Manip.A); // TODO: set ports to correct values
-    cargoIntakeBtn.whenPressed(new IntakeCargo(cargo));
+    cargoIntakeBtn.whenPressed(new IntakeCargo(cargo, lights));
     cargoEjectBtn = new JoystickButton(manipulator, Manip.B); // TODO: set ports to correct values
-    cargoEjectBtn.whenPressed(new EjectCargo(cargo));
+    cargoEjectBtn.whenPressed(new EjectCargo(cargo, lights));
```

Comment:
EjectCargo shouldn't need to be changed. Once IntakeCargo finishes the default lights command will resume which will reset the lights to indicate hatch.